components/table/table-col-sort-button.js

```diff
@@ -37,30 +37,40 @@ export class TableColSortButton extends FocusMixin(LitElement) {
 				align-items: center;
 				background-color: transparent;
 				border: none;
+				box-sizing: border-box;
 				color: inherit;
 				cursor: pointer;
 				display: inline-flex;
 				font-family: inherit;
 				font-size: inherit;
+				gap: 12px;
+				height: calc(var(--d2l-table-cell-height) + var(--d2l-sortable-button-height));
 				letter-spacing: inherit;
 				margin: 0;
-				padding: 0;
+				padding-inline-start: var(--d2l-sortable-button-padding);
 				text-decoration: none;
+				text-wrap: nowrap;
```

Comment:
You should confirm with Nicole what the desired overflow behaviour would be. We'd likely still want it to wrap if that's what it was doing before - it was more that the text in the visual diff now wrapping was a concern because it indicated something big changed.

Also it looks like [text-wrap](https://caniuse.com/?search=text-wrap) is pretty new and might not work as expected for all our users.

Reply:
I think last time the wrapping occured because of the added space between the text and icon